query/common.d.ts

```diff
@@ -170,7 +178,7 @@ export interface QueryVoid extends BaseQuery {
 /**
  * Union type for the different query types.
  */
-export type Query<SupportedMetadataType> = QueryBindings<SupportedMetadataType> | QueryBoolean | QueryQuads<SupportedMetadataType> | QueryVoid;
+export type Query<SupportedMetadataType> = QueryBindings<SupportedMetadataType> | QueryPaths | QueryBoolean | QueryQuads<SupportedMetadataType> | QueryVoid;
```

Comment:
Not sure if you need to follow QueryBindings and make it generic with `<SupportedMetadataType>`, but I guess we can just ignore that for now and see if it becomes needed.

Reply:
I think we can ignore the metadata object for now, it is not included in the "boolean" and "void" output types either. It is something that is passed back to the mediator for other actors to read, but we do not need this for v1.